src/Components/Display.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import CSS from "csstype";
+
+interface Props {
+  value: string;
+}
+
+const Display = ({ value }: Props) => {
```

Comment:
I think I'd just 
```
const Display = (props: { value: string }) => {
    return <div style={style}>{props.value}</div>;
};
```
without the interface

What you've done might be perfectly normal for all I know but usually (I think) interfaces are more about defining a type that'll see some re-use.

Reply:
for all I know this is perfectly normal in React + TS and at least you've done it consistently